### ISSUE TITLE
Refresh button width with another language

### ISSUE THREAD

**Issue by BeePerNet** [grafana/grafana#121039]
**Steps to reproduce**:

<img width="896" height="643" alt="Image" src="https://github.com/user-attachments/assets/4fe360fd-ab0b-41d3-ae07-cbf524c72961" />

**Actual Result**:

**Expected Result**
The "Actualiser" button width should be enough for his content.

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: Grafana v12.4.1 (46a02dc12a)
- Data source type & version:
- User OS & Browser: Windows 25H2 26200.8039 with Firefox 148.0.2
- Others:
